docs/contribute/new-snippets.md

```diff
@@ -2,18 +2,20 @@
 title: 'Code snippets contributions'
 ---
 
-Contributions to the snippets used in our documentation are welcome and greatly encouraged. This page outlines how to contribute to [Storybook's](https://github.com/storybookjs/storybook) code snippets. For instructions on working with the codebase, check the [code contributions](./code.md) page, and for documentation, check the [documentation contributions](./documentation-updates.md) page.
+Add or update the code snippets in the documentation. This page outlines how the code snippets are structured.
 
 ## Documented frameworks
 
-The Storybook documentation is highly versatile and constantly updated to solve everyday scenarios. Currently, it features examples for the following UI frameworks:
+Storybook maintains code snippets for a [variety of frameworks](./../api/frameworks-feature-support). We try to keep them up to date as framework APIs evolve. But it's tricky to keep track of every API change in every framework.
```

Comment:
@domyen when you're able, add the `.md` extension to this link, to allow the proper navigation in both GitHub and the website.

Reply:
Fixed with the latest commit. Marking this as solved